contrib/lib/src/space_helmet/policy.rs

```diff
@@ -0,0 +1,169 @@
+use rocket::http::uri::Uri;
+use rocket::http::Header;
+
+///[Referrer-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy) description.
+pub enum ReferrerPolicy {
+    NoReferrer,
+    NoReferrerWhenDowngrade,
+    Origin,
+    OriginWhenCrossOrigin,
+    SameOrigin,
+    StrictOrigin,
+    StrictOriginWhenCrossOrigin,
+    UnsafeUrl,
+}
+
+impl Default for ReferrerPolicy {
+    fn default() -> ReferrerPolicy {
+        ReferrerPolicy::NoReferrer
+    }
+}
+
+impl<'a, 'b> From<&'a ReferrerPolicy> for Header<'b> {
+    fn from(policy: &ReferrerPolicy) -> Header<'b> {
+        let policy_string = match policy {
```

Comment:
As far as I can see, all of these are static strings. Thus, there's no need to allocate a `String`. Simply using the raw `"string"`.